src/main/resources/templates/view-inventory.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
This template is where you can add your 'view inventory' data and call all of the methods/vars available associated with it

Reply:
This looks like the start of a new feature.  Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.